CalibCalorimetry/EcalTrivialCondModules/src/EcalLaserCondTools.cc

```diff
@@ -31,7 +31,9 @@ EcalLaserCondTools::EcalLaserCondTools(const edm::ParameterSet& ps)
       skipIov_(ps.getParameter<int>("skipIov")),
       nIovs_(ps.getParameter<int>("nIovs")),
       fromTime_(ps.getParameter<int>("fromTime")),
-      toTime_(ps.getParameter<int>("toTime")) {
+      toTime_(ps.getParameter<int>("toTime")),
+      minP_(ps.getParameter<double>("transparencyMin")),
+      maxP_(ps.getParameter<double>("transparencyMax")) {
```

Comment:
Adding new parameters means that current py files without the new parameters will crash in future CMSSW release and that the newer py files with the new parameters will crash in the older CMSSW release. I would suggest to introduce the `fillDescriptions` to set default values in order to preserve the compatibility between the releases (https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideConfigurationValidationAndHelp#An_Example)

Reply:
I don't understand. I expect that by "crash" you meant an exception, not a core dump.

The parameters are tracked parameters, so an exception will already be raised if they are missing in the configuration file.

Reply:
In addition this "fillDescription" is a source of inconsistency in the code. The new of the parameters will appear twice with the risk to update at one place and forget in another place or using a different spelling.  I have missed the rationally of this new method.